utils/protobuf-schema-utilities/src/test/proto/TestOrderingSyntax2.proto

```diff
@@ -8,6 +8,8 @@ message Address {
   optional string city = 3 [default = "Seattle"];
 
   repeated int32 samples = 6 [packed=true];
+
+  map<string, string> pastAddresses = 8;
```

Comment:
Can you please add a test case for map of nested types and enums?

Reply:
Also good to add multiple maps scattered at multiple locations.

Reply:
Can we add a test for map of primitive type to nested-type / enum? For Syntax 3, good to include multiple Proto 3 optionals as well.

Reply:
1. Added tests for enum and nested type as map value. 
2. According to the documentation, `map` fields and `repeated` fields cannot be added into `oneOf`.